test/sdk-benchmarks/src/main/java/software/amazon/awssdk/benchmark/enhanced/dynamodb/ItemFactory.java

```diff
@@ -0,0 +1,922 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.benchmark.enhanced.dynamodb;
+
+import com.amazonaws.util.ImmutableMapParameter;
+import java.lang.reflect.Method;
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+import software.amazon.awssdk.core.SdkBytes;
+
+abstract class ItemFactory<T> {
+    private static final String ALPHA = "abcdefghijklmnopqrstuvwxyz";
+
+    private static final Random RNG = new Random();
+
+    public final Map<String, T> tiny() {
+        return asItem(tinyBean());
+    }
+
+    public final Map<String, T> small() {
+        return asItem(smallBean());
+    }
+
+    public final Map<String, T> huge() {
+        return asItem(hugeBean());
+    }
+
+    public final Map<String, T> hugeFlat() {
+        return asItem(hugeBeanFlat());
+    }
+
+    public final TinyBean tinyBean() {
+        TinyBean b = new TinyBean();
+        b.setStringAttr(randomS());
+        return b;
+    }
+
+    public final SmallBean smallBean() {
+        SmallBean b = new SmallBean();
+        b.setStringAttr(randomS());
+        b.setBinaryAttr(randomBytes());
+        b.setListAttr(Arrays.asList(randomS(), randomS(), randomS()));
+        return b;
+    }
+
+    public final HugeBean hugeBean() {
+        HugeBean b = new HugeBean();
+        b.setHashKey(randomS());
+        b.setStringAttr(randomS());
+        b.setBinaryAttr(randomBytes());
+        b.setListAttr(Arrays.asList(randomS(),
+                randomS(),
+                randomS(),
+
```

Comment:
Why the seemingly random new lines? Also can we just for loop something here?

Reply:
The lines break them up in to powers of 2. I can change to using a loop